readme.md

```diff
@@ -452,6 +452,9 @@ print_r($insertedPKs);
 // )
 ```
 
+**Note:** due to the fact that `Faker` returns all the PKs inserted, the memory consumption will go up drasticly when you 
```

Comment:
What do you think about modifying to this:

```suggestion
**Note:** Due to the fact that `Faker` returns all the primary keys inserted, the memory consumption will go up drastically when you 
```

Also see https://www.merriam-webster.com/dictionary/drastically.